core/services/job/helpers_test.go

```diff
@@ -162,7 +163,7 @@ func makeOCRJobSpec(t *testing.T, transmitterAddress common.Address, b1, b2 stri
 
 	peerID := cltest.DefaultP2PPeerID
 	ocrKeyID := cltest.DefaultOCRKeyBundleID
-	jobSpecText := fmt.Sprintf(ocrJobSpecText, testutils.NewAddress().Hex(), peerID, ocrKeyID, transmitterAddress.Hex(), b1, b2)
+	jobSpecText := fmt.Sprintf(ocrJobSpecText, testutils.NewAddress().Hex(), testutils.FixtureChainID.String(), peerID, ocrKeyID, transmitterAddress.Hex(), b1, b2)
```

Comment:
nit/ You don't have to call `String()` with `%s`:
```suggestion
	jobSpecText := fmt.Sprintf(ocrJobSpecText, testutils.NewAddress().Hex(), testutils.FixtureChainID, peerID, ocrKeyID, transmitterAddress.Hex(), b1, b2)
```